web/src/main/webapp/WEB-INF/pages/application.jsp

```diff
@@ -19,21 +20,24 @@
             </h1>
         <div class="modal fade" id="modal" role="dialog">
             <div class="modal-dialog" >
-                <div class="modal-content">
+                <div class="modal-content" id="pop_up_content">
                     <div class="modal-header">
                     <h4 id="new_application_popup_title">Új jelentkezés</h4>
                     </div>
                     <div class="modal-body">
-                        <form class="form" role="form" method="POST" id="application-create-form" action="${new_application_popup}">
+                        <form class="form" role="form" method="POST" id="application-create-form" action="${new_application_popup}" onsubmit="return validateApplicationSourceAndPositionDropDowns()">
                             <input type="hidden" name="candidateId" id="candidateId" value="${candidateId}">
                             <div class="form-group" id="positionDiv">
                                 <label id="positionLabel" class="control-label col-lg-2 col-md-2 col-sm-2 text-right"
                                     for="drop"><spring:message
                                     code="application.create.position.label"/></label>
 
                                 <div class="selectContainer col-lg-4 col-md-4 col-sm-4" id="drop">
-                                    <select class="input form-control" name="position.id" id="position">
+                                    <spring:message code="application.popup.position.drop.down.default.value" var="i18n_position_drop_down_default_value"/>
+                                    <select class="input form-control" name="position.id" id="position" onchange = "positionValidator()">
```

Comment:
Register the onchange and onsubmit events by jQuery as the suggested js change will prevent you to access the functions directly from the markup.